app/controllers/orders_controller.rb

```diff
@@ -12,6 +12,7 @@ def index
   def show
     if Order.find(params[:id]).user == current_user
       @order = Order.find(params[:id])
+      @studio == current_user.studio
```

Comment:
Is the `==` a typo? Did you mean `=`? I don't see @studio/studio in the order show view anyway, so I'm not sure why it's in the controller.